src/unity/python/turicreate/toolkits/drawing_recognition/drawing_recognition.py

```diff
@@ -0,0 +1,480 @@
+            - 'recall'           : Recall score (macro average)
+            - 'f1_score'         : F1 score (macro average)
+            - 'confusion_matrix' : An SFrame with counts of possible
+            prediction/true label combinations.
+            - 'roc_curve'        : An SFrame containing information needed for an
+            ROC curve
+            
+            Returns
+            -------
+            out : dict
+            Dictionary of evaluation results where the key is the name of the
+            evaluation metric (e.g. `accuracy`) and the value is the evaluation
+            score.
+            
+            See Also
+            ----------
+            create, predict
+            
+            Examples
+            ----------
+            .. sourcecode:: python
+            
+            >>> results = model.evaluate(data)
+            >>> print results['accuracy']
+        """
+
+        predicted = self._predict_with_options(dataset)
+
+        target = _tc.SFrame({'label': _tc.SArray(dataset['label'])}) # fix this
+        
+        avail_metrics = ['accuracy', 'auc', 'precision', 'recall',
+                         'f1_score', 'confusion_matrix', 'roc_curve']
+
+        _tkutl._check_categorical_option_type(
+                        'metric', metric, avail_metrics + ['auto'])
+
+        if metric == 'auto':
+            metrics = avail_metrics
+        else:
+            metrics = [metric]
+
+        ret = {}
+        if 'accuracy' in metrics:
+            ret['accuracy'] = _evaluation.accuracy(target['label'], predicted['label'])
+        if 'auc' in metrics:
+            ret['auc'] = _evaluation.auc(target['label'], predicted['probability'], index_map=self._class_to_index)
+        if 'precision' in metrics:
+            ret['precision'] = _evaluation.precision(target['label'], predicted['label'])
+        if 'recall' in metrics:
+            ret['recall'] = _evaluation.recall(target['label'], predicted['label'])
+        if 'f1_score' in metrics:
+            ret['f1_score'] = _evaluation.f1_score(target['label'], predicted['label'])
+        if 'confusion_matrix' in metrics:
+            ret['confusion_matrix'] = _evaluation.confusion_matrix(target['label'], predicted['label'])
+        if 'roc_curve' in metrics:
+            ret['roc_curve'] = _evaluation.roc_curve(target['label'], predicted['probability'], index_map=self._class_to_index)
+        
+        return ret
+
+    def predict(self, dataset):
+        """ read this and fix:::
+            Predict object instances in an sframe of images.
+            
+            Parameters
+            ----------
+            dataset : SFrame | SArray | turicreate.Image
+            The images on which to perform object detection.
+            If dataset is an SFrame, it must have a column with the same name
+            as the feature column during training. Additional columns are
+            ignored.
+            
+            confidence_threshold : float
```

Comment:
How is this parameter passed?

Can the user get the class likelihoods?

Reply:
I think this is dead copy-paste that came in during `evaluate` and `predict` development. Will delete until we get better docs.